.github/workflows/helm.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Helm release'
+
+on:
+  push:
+    paths:
+      - 'helm-charts/l7mp-operator/Chart.yml'
+      - 'helm-charts/l7mp-ingress/Chart.yml'
+
+jobs:
+  release:
+    name: 'Helm release'
+    runs-on: ubuntu-latest
+    if: github.repository == 'l7mp/l7mp'
+    steps:
+      - name: 'checkout'
+        uses: actions/checkout@v2
+      - name: 'pyhton_client generate'
+        shell: bash
+        run: .python-client/generate
+      - name: 'k8s-operator build'
+        shell: bash
+        run: .k8s-operator/build
+      - name: 'build helm charts'
+        run: |
+          cd helm-charts
+          helm package l7mp-operator l7mp-ingress
+          cd ..
+      - name: 'update l7mp.io/charts'
+        run: |
+          git clone https://github.com/l7mp/l7mp.io.git
+          copy helm-charts/*.tgz l7mp.io/charts
+          helm repo index l7mp.io/charts/ --url https://l7mp.io/charts
+          git add l7mp.io
+          git commit -m "Automatic publish from l7mp/l7mp Helm release action"
+          git push l7mp.io master
```

Comment:
Commit message should be something like the following:
```
git commit -m "Update helm charts from l7mp/l7mp"  -m "(triggered by the 'Helm release' github action.)"
```
See, for example, https://chris.beams.io/posts/git-commit/

Reply:
Please see the changes in [40ee500](https://github.com/l7mp/l7mp/pull/99/commits/40ee500edcddf51e984fbea55d0cd4c255bbe4ba).